lib/src/components/NxIconDropdown/NxIconDropdown.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-button-helpers';
+@import '../../scss-shared/nx-dropdown-variables';
+
+.nx-icon-dropdown {
+  width: initial;
+
+  .nx-dropdown-menu {
+    right: 0;
+  }
+}
+
+.nx-icon-dropdown__toggle {
+  margin-right: 0;
```

Comment:
I worry about the inconsistencies with how this margin is implemented compared to the margins on `NxButton` and `NxSegmentedButton`.  Have you tried this component adjacent to other buttons to see if it works? Even if it does, I think being consistent with the other two components would be worthwhile.

Reply:
> I worry about the inconsistencies with how this margin is implemented compared to the margins on `NxButton` and `NxSegmentedButton`. Have you tried this component adjacent to other buttons to see if it works? Even if it does, I think being consistent with the other two components would be worthwhile.

As of this commit the spacing is consistent between `NxBtn` and this component: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/19c494acd418faef5fd053ba1cb9c39332ad06ee

Reply:
> > As of this commit the spacing is consistent between NxBtn and this component: [19c494a](https://github.com/sonatype/sonatype-react-shared-components/commit/19c494acd418faef5fd053ba1cb9c39332ad06ee)
> 
> It's not working correctly. I think we need examples of this type of button alongside other types of buttons just like `NxSegmentedButton` has.

Added a button bar example, fixed the margin so it's off the button and on the parent (so the button bar rules will work), and also added a button to the `nx-table` example. https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/ddd190459719ffbf050697aef3080d5ebda64b16